src/test/java/dev/openfga/sdk/api/auth/AccessTokenTest.java

```diff
@@ -25,16 +25,36 @@ class AccessTokenTest {
 
     private static Stream<Arguments> expTimeAndResults() {
         return Stream.of(
-                Arguments.of(Instant.now().plus(1, ChronoUnit.HOURS), true),
-                Arguments.of(Instant.now().minus(1, ChronoUnit.HOURS), false),
-                Arguments.of(Instant.now().minus(10, ChronoUnit.MINUTES), false),
-                Arguments.of(Instant.now().plus(10, ChronoUnit.MINUTES), true),
-                Arguments.of(Instant.now(), true));
+                Arguments.of("Expires in 1 hour should be valid", Instant.now().plus(1, ChronoUnit.HOURS), true),
+                Arguments.of(
+                        "Expires in 15 minutes should be valid", Instant.now().plus(15, ChronoUnit.MINUTES), true),
+                Arguments.of("No expiry value should be valid", null, true),
+                Arguments.of(
+                        "Expired 1 hour ago should not not be valid",
```

Comment:
```suggestion
                        "Expired 1 hour ago should not be valid",
```

Reply:
Fixed this and other test names in f98569c019551631f3f69b94e10d587f67fd6bc2